metadata-ingestion/src/datahub/ingestion/source/redshift/redshift.py

```diff
@@ -956,7 +965,9 @@ def generate_lineage(self, database: str) -> Iterable[MetadataWorkUnit]:
                 )
                 if lineage_info:
                     yield from gen_lineage(
-                        dataset_urn, lineage_info, self.config.incremental_lineage
+                        dataset_urn,
+                        lineage_info,
+                        False,  # incremental lineage generation is taken care by auto_incremental_lineage
```

Comment:
lets use a kwarg here and above

```suggestion
                        incremental_lineage=False,  # incremental lineage generation is taken care by auto_incremental_lineage
```